src/atoms/Blockie.tsx

```diff
@@ -0,0 +1,13 @@
+import makeBlockie from 'ethereum-blockies-base64';
+import type { FunctionComponent } from 'react';
+
+import type { ImageProps } from '.';
+import { Image } from '.';
```

Comment:
This can trigger circular dependency issues.
Prefer importing from the sibling file.